store/tikv/gc_worker.go

```diff
@@ -582,13 +707,48 @@ func (w *GCWorker) checkLeader() (bool, error) {
 	return false, nil
 }
 
-func (w *GCWorker) saveTime(key string, t time.Time) error {
-	err := w.saveValueToSysTable(key, t.Format(gcTimeFormat))
+func (w *GCWorker) saveSafePoint(key string, t uint64) error {
+	w.checkerLock.Lock()
+	defer w.checkerLock.Unlock()
```

Comment:
etcd client supports accessing concurrently. So it's better to remove the mutex. btw, don't forget to make sure `MockSafePointKV` is thread safe too.

Reply:
Thanks, I have revised accordingly.

Reply:
B.T.W, the mock client is not thread safe for concurrently accessing...